### PR TITLE
Use new group endpoint

### PR DESCRIPTION
**Pull request by Donatien26**

### CODE REVIEW

**Comment by Donatien26** [comment]
close #120

File: src/api/remote/groupApi.tsx [hunk 1]
```diff
@@ -21,9 +21,17 @@ export const getGroup = (
...
-				params: { application },
-			})
+			.get(
+				'/realms/' +
+					realm +
+					'/applications/' +
+					application +
+					'/groups/' +
+					group,
+				{
```
Comment (CChemin) [inline src/api/remote/groupApi.tsx:31]: params are not needed

File: src/api/remote/groupApi.tsx [hunk 2]
```diff
@@ -34,9 +42,17 @@ export const postGroup = (
...
-				params: { application },
-			})
+			.post(
+				'/realms/' +
+					realm +
+					'/applications/' +
+					application +
+					'/groups/',
+				group,
+				{
```
Comment (CChemin) [inline src/api/remote/groupApi.tsx:52]: params are not needed

File: src/api/remote/groupApi.tsx (hunk 1 above)
Comment (Donatien26) [reply src/api/remote/groupApi.tsx]: yep my bad

**Review by CChemin — APPROVED** [review]

**Pull request merged**